### PR TITLE
fix panel bug

### PR DESCRIPTION
**Pull request by tcm390**
allow the user to select objects in the world and have them show in the object panel.

Reference: https://github.com/webaverse/app/issues/3112

https://user-images.githubusercontent.com/60634884/172517162-5fb33315-7aab-435a-8aa8-8574d3175491.mp4

### CODE REVIEW

**Review by avaer — CHANGES_REQUESTED** [review]
This is close, but I think this update belongs in the `game.js` update loop?

**Comment by tcm390** [comment]
> move world object select to game.js

[move world object select to game.js](https://github.com/webaverse/app/pull/3114/commits/7af1fe7adf10680f646ebd85c1ba09f6bab5c386)

File: ai/lore/lore-model.js [hunk 1]
```diff
@@ -1 +1,2 @@
-export * from '../../lore/lore-model.js';
\ No newline at end of file
+// export * from '../../lore/lore-model.js';
+export * from 'http://localhost/lore-model.js';
```
[the remark this answers is not in the document]
Comment (tcm390) [reply ai/lore/lore-model.js]: Sorry @avaer but it does not in here https://github.com/webaverse/app/pull/3114/files

**Comment by lalalune** [comment]
Hello @tcm390 is this ready to be pulled in? Could you please resolve the packages/three conflict?
@alisaad673 has this been resviewed?

**Comment by lalalune** [comment]
@avaer reviewed this, works good

cool if we pull this?

image

**Comment by alisaad673** [comment]
Hey @lalalune  @tcm390  Sorry I missed the review request. 
I have tested this,

 the functionality seems to be working fine, except this one bug:
-While we select any object from world and edit it, After we close the world panel, there still is the pointer for selecting the objects also there is this position pointer too. We are unable to click into the world and unable to move camera direction. This  happens only when we try to close the panel with mouse. Doesn't happens when we close the panel with "Z" key.

https://user-images.githubusercontent.com/35926530/177983287-e120daf7-b1ae-40d3-ac03-1ca4c32a64ff.mp4

https://user-images.githubusercontent.com/35926530/177983775-23bfd5ca-6c31-4a8b-9781-86fd12fd36b6.mp4

**Comment by lalalune** [comment]
Close in favor of https://github.com/webaverse/app/pull/3262/

**Pull request closed without merging**